ontology/spdx-ontology.owl.xml

```diff
@@ -390,7 +355,8 @@ Known issues:</rdfs:comment>
             </owl:Class>
         </rdfs:range>
         <rdfs:comment xml:lang="en">Licensing information that was discovered directly in the subject file. This is also considered a declared license for the file.</rdfs:comment>
-        <ns:term_status>deprecated</ns:term_status>
+        <ns:term_status xml:lang="en">deprecated</ns:term_status>
+        <owl:deprecated rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</owl:deprecated>
     </owl:ObjectProperty>
```

Comment:
I'm manually merging these changes into a new PR - Just a note, this term is not deprecated, the term-status is in error.  I'll change this to `stable` in the new PR